src/features/wisdom/inlineSuggestions.ts

```diff
@@ -1,26 +1,22 @@
 import * as vscode from "vscode";
-
 import { v4 as uuidv4 } from "uuid";
 
 import { adjustInlineSuggestionIndent } from "../utils/wisdom";
 import { getCurrentUTCDateTime } from "../utils/dateTime";
 import { wisdomManager } from "../../extension";
-import { WisdomCommands } from "../../definitions/constants";
-import { resetKeyInput, getKeyInput } from "../../utils/keyInputUtils";
 import {
   CompletionResponseParams,
   InlineSuggestionEvent,
   CompletionRequestParams,
   UserAction,
 } from "../../definitions/wisdom";
 
-let suggestionId = "";
-let currentSuggestion = "";
-const taskRegexEp =
+const TASK_REGEX_EP =
   /(?<blank>\s*)(?<list>-\s*name\s*:\s*)(?<description>.*)(?<end>$)/;
 
+let suggestionId = "";
+let currentSuggestion = "";
 let inlineSuggestionData: InlineSuggestionEvent = {};
-let inlineSuggestionDisplayed = false;
```

Comment:
`inlineSuggestionDisplayed` and all the code that uses it are required, please revert back the changes for suggestion displayed logic

Reply:
Added.

Reply:
Why is that? Most likely you are not setting it properly at right place.